src/kixi/ckan/data.clj

```diff
@@ -6,12 +6,22 @@
             [clj-http.client       :as client]
             [clojure.tools.logging :as log]))
 
+(defn fix-map-key
+  "Takes string keys and format them before they
+  get turned into keywords."
+  [keys-string]
+  (-> keys-string
+      (clojure.string/lower-case)
+      (clojure.string/replace #"[ \n]" "_")
+      (clojure.string/replace #"[^A-Za-z0-9-_]" "")))
+
 (defn unparse
   "Takes DataStore data represented as JSON and turns it to clojure data structure."
   [data]
   (let [data-edn (-> data
-                     (json/parse-string)
-                     (get "result"))]
+                     (json/parse-string
+                      (fn [k] (keyword (fix-map-key k))))
```

Comment:
No need to change this but be aware that when composing fns like this sometimes it's clearer to do 
```
(comp keyword fix-map-key)
```